src/StatusQ/Components/StatusMessage.qml

```diff
@@ -35,156 +37,383 @@ Rectangle {
     property string errorLoadingImageText: ""
     property string audioMessageInfoText: ""
     property string pinnedMsgInfoText: ""
+    property var reactionIcons: [
+        "qrc:///src/assets/twemoji/svg/2764.svg", // heart
+        "qrc:///src/assets/twemoji/svg/1f44d.svg", // thumbs up
+        "qrc:///src/assets/twemoji/svg/1f44e.svg", // thumbs down
+        "qrc:///src/assets/twemoji/svg/1f606.svg", // laughing
+        "qrc:///src/assets/twemoji/svg/1f625.svg", // sad
+        "qrc:///src/assets/twemoji/svg/1f620.svg", // angry
+    ]
```

Comment:
```suggestion
    property var reactionIcons: [ ❤, 👍, 👎, 🤣, 😥, 😠]
```
I believe you could use unicode emojis here for better readability, then use `StatusQ.Emoji` to parse it.

Reply:
Ok, that makes sense, didn't know status-destkop uses emoji images different than those provided in twemoji.

Reply:
To be fair it still may be simplified:
```
    property var reactionIcons: [
        Emoji.iconSource("❤"),
        Emoji.iconSource("👍"),
        ...
        Emoji.iconSource("😠")
    ]
```
but that's not critical :D